test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/Search/SortTests.cs

```diff
@@ -90,13 +91,136 @@ public async Task GivenMoreThanTenPatients_WhenSearchedWithSortParamWithHyphen_T
             await ExecuteAndValidateBundle($"Patient?_tag={tag}&_sort=-_lastUpdated", false, patients.Reverse().Cast<Resource>().ToArray());
         }
 
+        // uncomment only when db cleanup happens on each run, otherwise the paging might cause expected resources to not arrive
```

Comment:
The idea of the tag is that your search would be scoped to it, so we do not depend on an empty database for each run.

Reply:
There are other tests which includes 'sort' + 'tag filter'. in this test I wanted to check the path of sorting alone. i.e. sorting without additional filters as it creates a different query. In order to achieve this the db must be clean....